src/houston/views/agreement.pug

```diff
@@ -4,14 +4,16 @@ block body
   div.grid
     div.two-thirds.scrollbox
       h2 Terms of Service
-      h5 Last updated December 8, 2016
+      h5 Last updated January 13, 2017
       p By accepting the Terms of this agreement you agree to be bound by the following Terms and Conditions of Use. If you do not agree with any these Terms, you are prohibited from making use of this website (the "Service") to distribute your application (the "Software").
       h5 Distribution
       p You agree that you own the rights to redistribute the Software through our Service, including commercial rights if you have marked the Software for monetization.
       p You agree to give elementary LLC. the rights to redistribute the Software through our Service.
       p elementary LLC. has the right to remove the Software from our Service and discontinue our Service at any time.
       h5 Monetization
-      p elementary LLC. will facilitate voluntary payment for the Software that you have marked for monetization at the cost of 30% of sales revenue, plus processing fees.
+      p elementary LLC. will facilitate voluntary payment for the Software that you have marked for monetization.
+      p Each payment is split between elementary LLC. and the you 30/70, with a minimum payment to elementary of $0.50 to cover distribution and payment processing.
+      p As an example, for a $10 payment, $3 goes to elementary, while $7 is paid out to you. For a $1 payment, $0.50 covers the cost of distribution and payment processing while $0.50 is paid out to you.
```

Comment:
I think it might be more clear this way since we're only mentioning the "distribution and payment processing" when we're also mentioning the 50 cent minimum.